Bellmanford_Algorithm/bellmanford.go

```diff
@@ -0,0 +1,429 @@
+package main
+
+import (
+	"container/list"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func main() {
+
+	
+
```

Comment:
Please use 4 space indentation instead of tabs

Reply:
remove the unnecessary lines